### PR TITLE
Fix broken link in ISSUE TEMPLATE

### PR DESCRIPTION
**Pull request by torn4dom4n**
## Overview
I fix a broken link in **ISSUE TEMPLATE**.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -29,4 +29,4 @@ If you'd like to make and test changes locally (and see how they would look if m
     - When successful, the script will initiate a local server at `http://127.0.0.1:4000/`.
 
 ### Questions?
-- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!
+- Leave a comment on this issue! Make sure to use @mentions if you want a specific person's attention!
```
Comment (brianamarie) [inline .github/ISSUE_TEMPLATE.md:32]:
```suggestion
- Leave a comment on this issue! Make sure to use `@mentions` if you want a specific person's attention!
```

If we use this in an issue template without a space, or without code blocks, we will be accidentally mentioning someone called `mentions` every time an issue is opened. 😊

**Review by brianamarie — CHANGES_REQUESTED** [review]
Hi @torn4dom4n! Thank you for making these updates.

I have suggested one change, and once that's implemented, this looks good to merge. ✨

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (torn4dom4n) [reply .github/ISSUE_TEMPLATE.md]: @brianamarie I got it. I changed with that your commit.

**Review by brianamarie — APPROVED** [review]
Fantastic. Thank you, @torn4dom4n!

**Pull request merged**